build_prod.py

```diff
@@ -210,11 +253,32 @@ def add_meta_layers(cfg):
             yocto_add_bblayer(cfg, bblayer)
 
 
-def build_init(uri, branch, xml_base_name):
+def build_init(uri, branch, xml_base_name, proposed):
+    if not branch:
+        branch = 'master'
+
     repo_init(uri, branch, xml_base_name)
     repo_sync()
+    c_dir = os.getcwd()
+    # go into prod layer directory to get the repo instance 
+    # and create the remote.
+    # (It is required to apply the pull request).
+    os.chdir('meta-xt-{}'.format(xml_base_name))
+    # Get url of prod-layer to find the pull requests
+    # The name of repository is known but organization - not
+    # The organization can be extracted from theurl of layer 'meta-xt-products'
+    # Assumption:the layers  products and prod belong 
+    # to the same organization    
+    url = "xen-troops/meta-xt-" + xml_base_name
```

Comment:
why `xen-troops` here? what if other organization is in place?

Reply:
at this moment - I do not have a solution in scope of the existed command line
but we can add additional command line parameter to process it.
does it work ?